1.9/api/access.md

```diff
@@ -0,0 +1,40 @@
+---
+post_title: Cluster Access
+nav_title: Cluster Access
+menu_order: 1
+---
+
+The cluster address may be obtained using the following methods:
+
+- Log into the DC/OS GUI and copy the scheme and domain name from the browser address bar.
+- Log into the DC/OS CLI and type `dcos config show core.dcos_url` to get the cluster URL.
+
+
+## API ports
+
+On the master nodes, Admin Router is accessible through standard ports: `80` (http) and `443` (https, if enabled).
+
+On the agent nodes, Admin Router Agent is accessible through port `61001` (http).
+
+
+## Agent node address
+
+The IP of specific agent nodes may be obtained using the following methods:
+
+- Log into the DC/OS GUI, navigate to the Nodes page, and copy the hostname of the desired node.
+- Log into the DC/OS CLI, list the nodes with `dcos node`, and copy the hostname of the desired node.
+
+To determine which agents are public agents, see [Finding a Public Agent IP](/docs/1.9/administration/locate-public-agent/).
+
+
+## Ingress
+
+In most production deployments, administrative access to the cluster should be routed through an external proxy to the DC/OS master nodes, distributing traffic load between the master nodes. For example, the default AWS templates configure an AWS Elastic Load Balancer.
+
+Master nodes and private agent nodes are usually not publicly accessible. For security reasons, ingress to these nodes should be controlled by a router or firewall. To manage the cluster, administrators and operators should use a VPN server inside the firewall, on the same network(s) as the DC/OS nodes. This way you can securely access the nodes directly from your workstation.
+
+Public agent nodes, on the other hand, are usually publicly accessible. Marathon-LB running on the public agent nodes can serve as reverse proxy and load balancer to applications running on the private agent nodes. For additional security, external load balancing may also be used, either to intermediate load balancers, applications on the public nodes, or directly to applications on the private nodes. If the public nodes are to be publicly accessible, firewalls should be configured to block access to all but the ports required for your applications.
```

Comment:
ugh, this reminds me the subject and tense are terrible inconsistent across this whole section.

I was originally trying to speak to the user of the API, making general statements about how an administrator may have set up the cluster. But now there's a few Yous mixed in that confuse the target audience.